### PR TITLE
Improve SAPOfflineTokenServicesCloud compatibility

### PR DESCRIPTION
**Pull request by hassler-d**
This PR improves the compatibility of the `SAPOfflineTokenServicesCloud` with the old implementation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file.
 
+## 2.7.6
+- [java-security] Improve compatibility of `SAPOfflineTokenServicesCloud`
+  - Now using client id (cid) from token for `clientId` attribute of `AuthorizationRequest`
```
Comment (nenaraab) [inline CHANGELOG.md:7]: Hi Daniel, this is a very technical description, which is fine with me, in case you also explain the symptom here :-)

**Comment by nenaraab** [comment]
Would you like to provide me the CHANGELOG later?

**Comment by hassler-d** [comment]
> Would you like to provide me the CHANGELOG later?

I have provided the CHANGELOG as part of this PR. You mean that I could extend it with an extended description and provide that later?

**Comment by nenaraab** [comment]
yes, imho the changelog is too technical and does not reflect the symptom... but you could do that later as well

**Comment by hassler-d** [comment]
> yes, imho the changelog is too technical and does not reflect the symptom... but you could do that later as well

I make the description less technical then and would push it into this PR if it is okay?

**Comment by nenaraab** [comment]
@hassler-d sure :-)

**Comment by hassler-d** [comment]
> @hassler-d sure :-)

I have updated the description. Can you check again?

**Pull request merged**